deps/chakrashim/lib/chakra_shim.js

```diff
@@ -214,12 +267,20 @@
       isPrepared = true;
       // Notify original Error object of this setter call. Without knowing
       // this Chakra runtime would reset stack at throw time.
-      Reflect_apply(oldStackDesc.set, e, [value]);
+      Reflect_apply(oldStackDesc.set, err, [value]);
```

Comment:
Don't understand `e -> err` change. (The function comment states) `e` was the Error instance. `err` can be anything, a normal object, used as `Error.captureStackTrace(err, ...)`. Also `oldStackDesc` was retrieved from `e` above. Cannot apply to `err`.